_posts/workshops/open-science/2016-09-07-intro-open-science.md

```diff
@@ -6,15 +6,16 @@ title: 'Introduction to Open Science Teaching Activity'
 excerpt: 'A hands-on activity where students review a project for readability,
 organization, etc and identify key elements that would make it more usable and
 readily reproducible.'
-dateCreated: '2016-10-25'
-modified: '2016-01-01'
-module-type: 'workshop'
+dateCreated: 2016-10-25
+modified: 2017-05-01
 module-title: 'Introduction to Open Science Teaching Activity'
 module-description: 'A hands-on activity where students review a project for
 readability, organization, etc and identify key elements that would
 make it more usable and readily reproducible.'
-permalink: /workshops/intro-open-science/
+permalink: /courses/intro-open-science/
```

Comment:
@lherwehe was this an intentional change? this is not a course it is a workshop

Reply:
it could be something else going on. i'll look at it locally.

Reply:
@lwasser - this was not an intentional change, i'm not sure how it happened ?!